src/components/ha-icon.ts

```diff
@@ -24,207 +24,17 @@ import {
-  "library-video": "play-box-multiple",
-  markdown: "language-markdown",
-  "markdown-outline": "language-markdown-outline",
-  "message-settings-variant": "message-cog",
-  "message-settings-variant-outline": "message-cog-outline",
-  "microsoft-dynamics": "microsoft-dynamics-365",
-  "network-router": "router-network",
-  office: "microsoft-office",
-  onedrive: "microsoft-onedrive",
-  onenote: "microsoft-onenote",
-  outlook: "microsoft-outlook",
-  playstation: "sony-playstation",
-  "periodic-table-co": "molecule-co",
-  "periodic-table-co2": "molecule-co2",
-  pot: "pot-steam",
-  ruby: "language-ruby",
-  sailing: "sail-boat",
-  scooter: "human-scooter",
-  settings: "cog",
-  "settings-box": "cog-box",
-  "settings-outline": "cog-outline",
-  "settings-transfer": "cog-transfer",
-  "settings-transfer-outline": "cog-transfer-outline",
-  "shield-refresh": "shield-sync",
-  "shield-refresh-outline": "shield-sync-outline",
-  "sort-alphabetical": "sort-alphabetical-variant",
-  "sort-alphabetical-ascending": "sort-alphabetical-ascending-variant",
-  "sort-alphabetical-descending": "sort-alphabetical-descending-variant",
-  "sort-numeric": "sort-numeric-variant",
-  "star-half": "star-half-full",
-  storefront: "storefront-outline",
-  timer: "timer-outline",
-  "timer-off": "timer-off-outline",
-  towing: "tow-truck",
-  voice: "account-voice",
-  "wall-sconce-variant": "wall-sconce-round-variant",
-  wii: "nintendo-wii",
-  wiiu: "nintendo-wiiu",
-  windows: "microsoft-windows",
-  "windows-classic": "microsoft-windows-classic",
-  worker: "account-hard-hat",
-  xbox: "microsoft-xbox",
-  "xbox-controller": "microsoft-xbox-controller",
-  "xbox-controller-battery-alert": "microsoft-xbox-controller-battery-alert",
-  "xbox-controller-battery-charging":
-    "microsoft-xbox-controller-battery-charging",
-  "xbox-controller-battery-empty": "microsoft-xbox-controller-battery-empty",
-  "xbox-controller-battery-full": "microsoft-xbox-controller-battery-full",
-  "xbox-controller-battery-low": "microsoft-xbox-controller-battery-low",
-  "xbox-controller-battery-medium": "microsoft-xbox-controller-battery-medium",
-  "xbox-controller-battery-unknown":
-    "microsoft-xbox-controller-battery-unknown",
-  "xbox-controller-menu": "microsoft-xbox-controller-menu",
-  "xbox-controller-off": "microsoft-xbox-controller-off",
-  "xbox-controller-view": "microsoft-xbox-controller-view",
-  yammer: "microsoft-yammer",
-  "youtube-creator-studio": "youtube-studio",
-  "selection-mutliple": "selection-multiple",
-  textarea: "form-textarea",
-  textbox: "form-textbox",
-  "textbox-lock": "form-textbox-lock",
-  "textbox-password": "form-textbox-password",
-  "syllabary-katakana-half-width": "syllabary-katakana-halfwidth",
-  "visual-studio-code": "microsoft-visual-studio-code",
-  "visual-studio": "microsoft-visual-studio",
-};
+interface DeprecatedIcon {
+  [key: string]: {
+    change: "removed" | "renamed";
```

Comment:
```suggestion
    removed: boolean;
```

Reply:
Used renamed instead since I actually check for that, and have adjusted the rest accordingly.